com.avaloq.tools.ddk.xtext.format/src/com/avaloq/tools/ddk/xtext/format/resource/FormatResourceDescriptionStrategy.java

```diff
@@ -88,11 +88,12 @@ public boolean createEObjectDescriptions(final EObject eObject, final IAcceptor<
    * @return true if description was created correctly
    */
   public boolean createDescriptionsForNonXbaseFormalParameters(final EObject eObject, final IAcceptor<IEObjectDescription> acceptor) {
-    if (!(eObject.toString().contains("self") || eObject.toString().contains("config") || eObject.toString().contains("elements")
-        || eObject.toString().contains("object"))) {
-      return super.createEObjectDescriptions(eObject, acceptor);
-    }
-    return false;
+    return shouldCreateDescriptionForObject(eObject) && super.createEObjectDescriptions(eObject, acceptor);
+  }
+
+  private boolean shouldCreateDescriptionForObject(final EObject eObject) {
+    return (!(eObject.toString().contains("self") || eObject.toString().contains("config") || eObject.toString().contains("elements")

```

Comment:
could we refactor `eObject.toString()` to a variable?

Reply:
ok. thank you.